.travis/custom_pylint.py

```diff
@@ -1,7 +1,7 @@
 #                                                         -*- coding: utf-8 -*-
 # SPDX-License-Identifier: MIT
 #
-# Copyright (c) 2019 Red Hat, Inc.
+# Copyright (c) 2019 - 2020 Red Hat, Inc.
```

Comment:
I think it is more common not to use space, i.e. write it as `2019-2020`

Reply:
Changed.